cypress/integration/actions.feature

```diff
@@ -15,8 +15,7 @@ Feature: Users can approve, accept, unapprove, and unaccept data
         When the user clicks the Approve button
         Then a modal confirmation dialog is displayed
         When the confirmation button is clicked
-        Then a circular loader is rendered
```

Comment:
No longer necessary because the data's cached.

Reply:
I think rather than changing the test we should tweak the implementation. I don't think in this case showing stale data is appropriate. I will work on that.

Reply:
Ah ok, in that case we can just switch the use of `{ loading } = useDataQuery(query)` to `{ fetching } = useDataQuery(query)`, that will keep the old behaviour intact.